experimental/packages/opentelemetry-proto/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@opentelemetry/proto",
+  "private": true,
+  "version": "0.27.0",
+  "description": "OpenTelemetry Collector Metrics Exporter allows user to send collected metrics to the OpenTelemetry Collector",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "compile": "tsc --build",
+    "generate": "./compile_proto.sh",
```

Comment:
Shell scripts wont work on windows. Is it planned to commit the generated files and use this only in case of updates?

Reply:
This was just to get moving quickly locally. I will make it platform agnostic before merge